device-code-flow/README.md

```diff
@@ -0,0 +1,128 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- PowerShell
+page_type: sample
+name: "PowerShell console application that makes a request to the Graph API via the Device Code flow"
+description: "This PowerShell console application uses the device code flow for authentication and then makes a request to Microsoft Graph for the user's profile data."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-app-device-code-powershell
+---
+-->
+
+# PowerShell | console | user sign-in, protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This PowerShell console application authenticates a user via the device code flow, and then makes a request to the Graph API as the authenticated user. The response to the request is printed to the terminal.
+
+```console
+.\app.ps1
+
+To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code XXXXXXXXX to authenticate.
+
+{
+  @odata.context    : https://graph.microsoft.com/v1.0/$metadata#users/$entity
+  businessPhones    : {+1 (999) 5551001}
+  displayName       : Contoso Employee
+  givenName         : Contoso
+  jobTitle          : Worker
+  mail              : cemployee@contoso.com
+  mobilePhone       : 1 999-555-1001
+  officeLocation    : Contoso Plaza/F30
+  preferredLanguage :
+  surname           : Employee
+  userPrincipalName : contoso_employee@contoso.com
+  id                : 5a55a5dd-1a60-3989-413b-116087b0b941
+}
+```
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- PowerShell
+- MSAL.NET
```

Comment:
Im personally not clear whether a package/module is a pre-req. I've seen some readme template that has the packages installation as part of the `Run the app` section.

Reply:
Thank you for this.  I was not sure whether to include the step for installing MSAL.NET.  I can add it, it's a one line operation.